From c22e6793b94df6bef9e51e01ae8248b3d09af8a3 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Thu, 24 Mar 2022 15:22:55 -0300
Subject: [PATCH 029/163] stdlib: Fix tst-rand48.c printf types

Checked on x86_64-linux-gnu and i686-linux-gnu.

(cherry picked from commit 3ff447f7dfcb79740e30f6247c1d4d5344244a35)
---
 stdlib/tst-rand48.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/stdlib/tst-rand48.c b/stdlib/tst-rand48.c
index 973c62e1eb..ec8195c6d5 100644
--- a/stdlib/tst-rand48.c
+++ b/stdlib/tst-rand48.c
@@ -26,7 +26,7 @@ do_test (void)
   if (xsp[0] != 0x330e || xsp[1] != 0x5432 || xsp[2] != 0x9876)
     {
       puts ("srand48(0x98765432) didn't set correct value");
-      printf ("  expected: { %04hx, %04hx, %04hx }\n", 0x330e, 0x5432, 0x9876);
+      printf ("  expected: { %04x, %04x, %04x }\n", 0x330e, 0x5432, 0x9876);
       printf ("  seen:     { %04hx, %04hx, %04hx }\n", xsp[0], xsp[1], xsp[2]);
       result = 1;
     }
@@ -76,7 +76,7 @@ do_test (void)
   if (xsp[0] != 0x62f2 || xsp[1] != 0xf474 || xsp[2] != 0x9e88)
     {
       puts ("seed48() did not install the values correctly");
-      printf ("  expected: { %04hx, %04hx, %04hx }\n", 0x62f2, 0xf474, 0x9e88);
+      printf ("  expected: { %04x, %04x, %04x }\n", 0x62f2, 0xf474, 0x9e88);
       printf ("  seen:     { %04hx, %04hx, %04hx }\n", xsp[0], xsp[1], xsp[2]);
       result = 1;
     }
@@ -215,7 +215,7 @@ do_test (void)
   if (xsp[0] != 0x0637 || xsp[1] != 0x7acd || xsp[2] != 0xdbec)
     {
       puts ("seed48() did not install the values correctly");
-      printf ("  expected: { %04hx, %04hx, %04hx }\n", 0x0637, 0x7acd, 0xdbec);
+      printf ("  expected: { %04x, %04x, %04x }\n", 0x0637, 0x7acd, 0xdbec);
       printf ("  seen:     { %04hx, %04hx, %04hx }\n", xsp[0], xsp[1], xsp[2]);
       result = 1;
     }
-- 
2.37.2

